src/Models/Elevation.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace Firesphere\SolrSearch\Models;
+
+use SilverStripe\ORM\DataObject;
+
+class Elevation extends DataObject
+{
+    private static $table_name = 'Elevation';
+
+    private static $db = [
+        'Keyword' => 'Varchar(255)',
+    ];
+
+    private static $many_many = [
+        'Items' => ElevatedItem::class,
+    ];
+
+    private static $summary_fields = ['ID', 'Keyword'];
```

Comment:
As above, it would be good to list them out instead of using as a string.

Also, I'd add `Items.Count`, so people can see how many items are elevated by this keyword without having to look deeper.